packages/internals/src/engine-commands/getDmmf.ts

```diff
@@ -99,6 +110,109 @@ export async function getDMMF(options: GetDMMFOptions): Promise<DMMF.Document> {
   return dmmf
 }
 
+async function getDmmfWasm(options: GetDMMFOptions) {
+  const debugErrorType = createDebugErrorType(debug, 'getDmmfWasm')
+  debug(`Using getDmmf Wasm`)
+
+  const dmmfEither = pipe(
+    E.tryCatch(
+      () => {
+        if (process.env.FORCE_PANIC_QUERY_ENGINE_GET_DMMF) {
+          debug('Triggering a Rust panic...')
+          prismaFmt.debug_panic()
+        }
+
+        const params = JSON.stringify({
+          prismaSchema: options.datamodel,
+        })
+
+        const data = prismaFmt.get_dmmf(params)
```

Comment:
Time to rename `prismaFmt`, at least this variable here already maybe?

Reply:
I'd gladly do that in another PR, where I'm also going to remove code that's going to be made redundant by this PR.
Does `prismaWasm` sound good? Should we rename the package name as well (from `@prisma/prisma-fmt-wasm` to `@prisma/psl-utils-wasm`)?

Reply:
I think we had a plan to rename it to `prisma-schema-wasm` or something like that, right @tomhoule?

Reply:
yes there's an issue for it I think. That should be done separately.

Reply:
(Ignore my comment then, `prismaFmt` still refers to a package called `prisma-fmt-wasm` - so this is correct for now.